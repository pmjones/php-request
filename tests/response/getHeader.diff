003+ string(0) ""
003- NULL