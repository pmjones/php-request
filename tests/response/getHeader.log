
---- EXPECTED OUTPUT
string(3) "baz"
string(3) "baz"
NULL
---- ACTUAL OUTPUT
string(3) "baz"
string(3) "baz"
string(0) ""
---- FAILED
