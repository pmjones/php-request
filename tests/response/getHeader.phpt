--TEST--
SapiResponse::getHeader
--FILE--
<?php
$response = new SapiResponse();
$response->setHeader('Foo-Bar', 'baz');
var_dump($response->getHeader('Foo-Bar'));
var_dump($response->getHeader('foo-bar'));
var_dump($response->getHeader('dib-zim'));
--EXPECT--
string(3) "baz"
string(3) "baz"
NULL
