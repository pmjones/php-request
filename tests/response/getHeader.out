string(3) "baz"
string(3) "baz"
string(0) ""